static/elements/chromedash-metrics.js

```diff
@@ -31,6 +29,53 @@ class ChromedashMetrics extends LitElement {
     };
   }
 
+
+  static get styles() {
+    return [
+      SHARED_STYLES,
+      css`
+      :host {
+        display: block;
+        flex: 1;
+        padding: 1px;
+      }
+
+      #stack-rank-list {
```

Comment:
nit: can we change to a class. (if easy to change)

Reply:
Done.